app/styles/ui/check-runs/_ci-check-run-list.scss

```diff
@@ -1,6 +1,7 @@
 .ci-check-run-list {
   overflow-y: auto;
   overflow-x: hidden;
+  scroll-padding-top: 35px;
```

Comment:
Found out that the scroll-margin-top on the job stop lists for some reason doesn't work if the first failed step is in the first workflow action group... This makes it work... but unfortunately, does push the non-first ones down a step (making a non-failed step visible right before it). I may revisit this to see if there is a solution to make it uniform (first failed step being always the first one seen). But at least this way the first failed step is visible in both cases.